testqa

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
What is the purpose of this top level file?

Reply:
For discussions on TestQA ADR topics like what might have been done for TAF decision.